doc/2/framework/classes/kuzzle-request/get-search-body/index.md

```diff
@@ -0,0 +1,33 @@
+---
+code: true
+type: page
+title: getSearchBody
+description: KuzzleRequest class getSearchBody() method
+---
+
+# getSearchBody
+
+<SinceBadge version="2.11.0" />
+
+Returns the search body.
+
+With the HTTP protocol, the search body will be fetch depending on the verb:
+  - `GET`: extract the `searchBody` query string argument and parse it
+  - `POST`: return the request body
+
+Typically the search body contains keywords used by Elasticsearch to perform queries like `query`, `sort`, `aggregations` etc.
```

Comment:
I don't understand the purpose of this sentence.

Reply:
People are often confused about what is inside the search body (one of our client for example)